app/controllers/admin/service_orders_controller.rb

```diff
@@ -3,9 +3,9 @@ class Admin::ServiceOrdersController < Admin::AdminController
   layout 'admin'
 
   def index
-    @service = ServiceOrder.order("id ASC")
+    @service = ServiceOrder.order("id ASC").page params[:page]
```

Comment:
There are **3** things need to be improved here:

1. ``order(id: :asc)`` instead
2. should move to scope instead
So, you should change it to something similar to

``ServiceOrder.page(page_param)``

And define a scope in model ``ServiceOrder``

``default_scope #st there``

3. Check our convention about call **params** variable

Suggest to create a method call ``page_param`` in private scrope.